static/src/js/components/DraftList/DraftList.jsx

```diff
@@ -59,63 +71,181 @@ const DraftList = ({ draftType }) => {
+  //                   Icon={FaFileDownload}
+  //                   onClick={() => handleDownloadClick(conceptId)}
+  //                   variant="secondary"
+  //                   size="sm"
+  //                 >
+  //                   Download JSON
+  //                 </Button>
+  //               </div>
+  //             )
+  //           }
+  //         ]
+  //     }
+  //   )
+  // })
+  // )
+  const buildPrimaryEllipsisLink = useCallback((originalCellData, rowData) => {
+    const { conceptId } = rowData
+
+    let cellData = originalCellData
+
+    if (!cellData && draftType === conceptIdTypes.C) cellData = '<Blank Short Name>'
+    if (!cellData) cellData = '<Blank Name>'
 
     return (
-      {
-        key: conceptId,
-        cells:
-          [
-            {
-              value:
-              (
-                <Link to={draftLink}>
-                  {Name || ShortName || '<Blank Name>'}
-                </Link>
-              )
-            },
-            {
-              value:
-              (
-                LongName || EntryTitle || '<Untitled Record>'
-              )
-            },
-            {
-              value:
-              (
-                new Date(revisionDate).toISOString().split('T')[0]
-              )
-            },
-            {
-              value:
-              (
-                <div className="d-flex">
-                  <Button
-                    className="d-flex"
-                    Icon={FaFileDownload}
-                    onClick={() => handleDownloadClick(conceptId)}
-                    variant="secondary"
-                    size="sm"
-                  >
-                    Download JSON
-                  </Button>
-                </div>
-              )
-            }
-          ]
-      }
+      <EllipsisLink to={`/drafts/${`${paramDraftType.toLowerCase()}`}/${conceptId}`}>
+        {cellData}
+      </EllipsisLink>
     )
-  })
-  )
+  }, [])
+
+  const buildEllipsisTextCell = useCallback((originalCellData) => {
+    let cellData = originalCellData
+
+    if (!cellData && draftType === conceptIdTypes.C) cellData = '<Blank Entry Title>'
+    if (!cellData) cellData = '<Blank Long Name>'
+
+    return (
+      <EllipsisText>
+        {cellData}
+      </EllipsisText>
+    )
+  }, [])
+
+  const buildActionsCell = useCallback((cellData, rowData) => {
+    const { conceptId } = rowData
+
+    return (
+      <div className="d-flex">
+        <Button
+          className="d-flex"
+          Icon={FaFileDownload}
+          iconTitle="Document with an arrow pointing down"
+          onClick={() => handleDownloadClick(conceptId)}
+          variant="secondary"
+          size="sm"
+        >
+          Download JSON
+        </Button>
+      </div>
+    )
+  }, [])
+
+  const [columns, setColumns] = useState([
+    {
+      dataKey: 'revisionDate',
+      title: 'Last Modified',
+      className: 'col-auto',
+      dataAccessorFn: (cellData) => cellData.split('T')[0]
```

Comment:
You may have gotten this name from react-table, but I think maybe cellRendererFn might be a little more descriptive?